code/script.js

```diff
@@ -29,23 +37,246 @@ const showMessage = (message, sender) => {
+
+  
+  let appointmentBtn = document.getElementById('appointmentBtn');
+  appointmentBtn.addEventListener('click', () => bookAppointment(firstName))
+
+  let openingHoursBtn = document.getElementById('openingHoursBtn');
+  openingHoursBtn.addEventListener('click', () => openingHours(firstName))
+
+  let styleAdviceBtn = document.getElementById('styleAdviceBtn');
+  styleAdviceBtn.addEventListener('click', () => styleAdvice(firstName))
+}
+
+
+//If user clicks on button appointmentBtn, this runs:
+const bookAppointment = (firstName) => {
+  userReply(`I'd like to make an appointment`);
+  setTimeout(() => botReply(`Great! Please select the treatment you would like to recieve below.`), 500)
+  
+  inputWrapper.innerHTML =
+        `<select id="selectTreatment">
+          <option value selected disabled>Select a treatment below:</option>
+          <option value="Hair cutting">Hair cutting</option>
+          <option value="Hair coloring">Hair coloring</option>
+          <option value="Hair styling">Hair styling</option>
+        </select>
+        <button class="selectBtn" id="selectTreatmentBtn" type="submit">Send</button>
+        `;
+  
+        let selectTreatmentBtn = document.getElementById('selectTreatmentBtn');
+        let selectedTreatment = document.getElementById('selectTreatment');
+        selectTreatmentBtn.addEventListener('click', () => bookDate(selectedTreatment, firstName))
 }
 
-// Set up your eventlisteners here
+const bookDate = (selectedTreatment, firstName) => {
+  //console.log(`${selectTreatment.value}`)
+  userReply(`${selectTreatment.value}`);
+  botReply(`Excellent choice ${firstName}! Which one of our available time slots would suit you best?`);
+
+  inputWrapper.innerHTML =
+        `<select id="selectDate">
+          <option value selected disabled>Available time slots:</option>
+          <option value="Today at 12.30">Today at 12.30</option>
+          <option value="Today at 16.45">Today at 16.45</option>
+          <option value="Tomorrow at 10.00">Tomorrow at 10.00</option>
+          <option value="Tomorrow at 13.30">Tomorrow at 13.30</option>
+          <option value="Tomorrow at 15.30">Tomorrow at 15.30</option>
+        </select>
+        <button class="selectBtn" id="selectDateBtn" type="submit">Send</button>
+        `;
+
+        let selectDateBtn = document.getElementById('selectDateBtn');
+        let selectedDate = document.getElementById('selectDate');
+        selectDateBtn.addEventListener('click', () => confirmBooking(selectedTreatment, selectedDate, firstName))
+
+}
+
+const confirmBooking = (selectedTreatment, selectedDate, firstName, ) => {
+    //Transform selection strings to lowercase (for use within a sentence)
+      const selectedTreatmentValue = selectedTreatment.value;
+      const selectDateValue = selectedDate.value;
+      const lowerCaseTreatment = selectedTreatmentValue.toLowerCase();
+      const lowerCaseDate = selectDateValue.toLowerCase();
```

Comment:
Do you really need to store  selectedTreatment.value or can you jump directly to const lowerCaseTreatment? (Same for date) - Nice naming of variables by the way.